.changeset/swift-hotels-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+"app-builder-lib": patch
+---
+
+skip circular deps
```

Comment:
Please make this more descriptive of a changeset since this is published to our changelog

Reply:
Fixed.